animations/index.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: page
+title: Animations in Flutter
+permalink: /animations/
+---
+
+* TOC Placeholder
+{:toc}
+
+Well-designed animations makes a UI feel more intuitive,
+contribute to the slick look and feel of a polished app,
+and improve the user experience. Flutter's animation support
+makes it easy to implement a variety of animation types.
+Many widgets, especially
+[Material Design widgets](https://flutter.io/widgets/material/),
+come with the standard motion effects defined in their design spec,
+but it's also possible to customize these effects.
+
+The following resources are a good place to start learning the Flutter
+animation framework. Each of these docs shows, step by step, how
+to write animation code.
+Note that more docs are in the works on how to implement common design patterns,
+such as shared element transitions, and physics-based animations.
+If you have a specific request, please
+[file an issue](https://github.com/flutter/flutter/issues).
+
+* [Tutorial: Animations in Flutter](/tutorials/animation/) <img src="/images/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
```

Comment:
The alt text on this image is redundant with the text that follows it. We should have one or the other, not both.

Reply:
Travis fails unless there's ALT text, or I would have left it off. The visible text is important to clarify the meaning of the icon.

Reply:
We should fix Travis. It's invalid HTML to have redundant alt="" text. See https://html.spec.whatwg.org/multipage/images.html#alt